Products/SQLAlchemyDA/da.py

```diff
@@ -451,10 +451,10 @@ def manage_doQuery(self, query):
     security.declareProtected(view_management_screens, 'manage_formatItem')
     def manage_formatItem(self, s):
         """ used by query form """
-        if isinstance(s, unicode):
-            return s
-        if isinstance(s, str):
-            return unicode(s, self.encoding, 'ignore')
+        # if isinstance(s, unicode):
+        #    return s
+        # if isinstance(s, str):
+        #    return unicode(s, self.encoding, 'ignore')
```

Comment:
Don't just comment this out. Look at the intent and go from there. The intent is to safely convert bytes values into unencoded string values (unicode in Python 2, native string in Python 3).